operations/helm/charts/grafana-agent/values.yaml

```diff
@@ -254,6 +254,8 @@ service:
   enabled: true
   # -- Service type
   type: ClusterIP
+  # -- NodePort port
+  nodePort: 31128
```

Comment:
Perhaps the comment could indicate this only takes effect when type is `NodePort`.

Reply:
Absolutely.
I just fixed it.